uctMaths/competition/compadmin.py

```diff
@@ -651,7 +651,7 @@ def makeCertificates(students, assigned_school):
             for student in students:
                 award = student.award            
 
-                if award in certs:
+                if award in certs or award == '':
```

Comment:
Ok, it turns out that I fixed the wrong issue, it saves award as None for absences once awards are assigned, which would return a participation certificate, so I should maybe fix that still

Reply:
So it looks like in other places in the code we use `student.score > 0` to check for non-ABSENSE. Which is slightly strange since it is possible (but really difficult) to get 0 for the UCT maths competition.

Reply:
For me score is None for absences, but maybe they upload a result of 0 to absences

Reply:
Isn't it also `None` for participants who are present but didn't get an award?

Reply:
Yes, but they have a non-None score, whereas for me absentees have both as none